OptimoveSDK/optimove-sdk/src/main/AndroidManifest.xml

```diff
@@ -11,25 +11,19 @@
         android:dataExtractionRules="@xml/optimove_backup_rules_api31"
         android:label="@string/app_name"
         android:supportsRtl="true">
-        <service
-            android:name=".optipush.events_dispatch_service.NotificationOpenedEventDispatchService"
-            android:enabled="true"
+        <provider
+            android:name=".optimobile.OptimobileInitProvider"
+            android:authorities="${applicationId}.optimobileinitprovider"
+            android:exported="false" />
+        <activity
+            android:name=".optimobile.PushOpenInvisibleActivity"
+            android:noHistory="true"
+            android:excludeFromRecents="true"
+            android:theme="@android:style/Theme.Translucent"
             android:exported="false" />
         <service
             android:name=".main.app_update_listener.AppUpdateService"
             android:exported="false"
             android:permission="android.permission.BIND_JOB_SERVICE" />
-        <receiver
-            android:name=".main.app_update_listener.AppUpdateReceiver"
```

Comment:
What was the purpose of this receiver, is it relevant for Optimobile?

Reply:
The purpose was to silently fetch the configuration file when an update happens

Reply:
@KonstantinOptimove so do we need to retain it or is it no l longer relevant?